modules/nf-core/toulligqc/tests/main.nf.test

```diff
@@ -0,0 +1,87 @@
+nextflow_process {
+
+    name "Test Process TOULLIGQC"
+    script "../main.nf"
+    process "TOULLIGQC"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "toulligqc"
+
+    test("sarscov2 - nanopore sequencing_summary") {
+        when {
+            process {
+                """
+                input[0] = [
+                    file(params.test_data['sarscov2']['nanopore']['test2_sequencing_summary'], checkIfExists: true)
+                    ]
+                input[1] = [
```

Comment:
```suggestion
                input[1] = [
                  [],
                  []
```

And this for optional inputs :)

Reply:
Ah of course!! I forgot to add these to my test, that makes sense thank you!!